frontend/src/components/navbar.js

```diff
@@ -1,57 +1,73 @@
-import React, { useState } from 'react';
-import { Navbar, Form, Button, NavbarToggler, NavbarBrand, Nav, NavItem, NavLink, InputGroup, Input } from 'reactstrap';
-import { BrowserRouter as Router, Switch, Route} from 'react-router-dom';
+import React from 'react';
+import 'antd/dist/antd.css';
+import { Menu } from 'antd';
+import { QuestionOutlined, GithubOutlined, HomeOutlined, UserOutlined, LoginOutlined, PlusOutlined, AppstoreAddOutlined } from '@ant-design/icons';
 
+import { BrowserRouter as Router, Switch, Route} from 'react-router-dom';
+import '../styles/navbar.css' 
 import homePage from './home'
-
+import concertPlaylist from './concertPlaylist'
```

Comment:
We have to remove this as well, if not we get an error saying that it cannot find concertPlaylist

Reply:
Done.